README.md

```diff
@@ -239,11 +245,11 @@ gcloud projects add-iam-policy-binding ${PROJECT} \
 Create a secret key containing a GCP private key for the service account
 
 ```
-gcloud iam service-accounts keys create ~/tmp/key.json \
+KEY_FILE=<path to key>
+gcloud iam service-accounts keys create ${KEY_FILE} \
     	--iam-account ${SERVICE_ACCOUNT}@${PROJECT}.iam.gserviceaccount.com
 kubectl create secret generic kubeflow-testing-credentials \
-    --namespace=kubeflow-test-infra --from-file=`echo ~/tmp/key.json`
-rm ~/tmp/key.json
+    --namespace=kubeflow-test-infra --from-file=key.json=${KEY_FILE}
```

Comment:
additional =key.json?

Reply:
This one is correct. The key refers to how the file will be referenced inside the K8s configmap. We always want to call it key.json so we can refer to it in other components.